application/views/surveyAdministration/tabCopy_view.php

```diff
@@ -61,7 +61,7 @@
                 <div class="ls-flex-column col-md-4">
                     <!-- Convert resource links -->
                     <div class="form-group">
-                        <label class=" control-label" for='copysurveytranslinksfields'><?php echo  eT("Convert resource links and expression fields?"); ?> </label>
+                        <label class=" control-label" for='copysurveytranslinksfields'><?php echo  eT("Copy survey resource files and adapt links"); ?> </label>
```

Comment:
No way to convert link *without** copy directory ? Why not, but maybe need an extra option ?